src/patterns/fill-in-a-form/index.md.njk

```diff
@@ -111,6 +111,35 @@ Use the Details component to make your page easier to scan, but don’t hide hel
 
 Make sure the link text is written so that users can quickly work out if they need to click on it.
 
+
+
+
+### Using progress indicators
+
+Start by testing your form without a progress indicator to see if it’s simple enough that users don’t need them.
+
+Try improving the order, type or number of questions before adding a progress indicator. If people still have difficulty, try adding a simple step or question indicator like this one.
+
+{{ example({group: 'patterns', item: 'fill-in-a-form', example: 'progress', html: true, nunjucks: true, open: false}) }}
```

Comment:
This example doesn't contain any Nunjucks syntax. Shall we remove the Nunjucks tab for it?

Reply:
There is a bug on the tabs right now that needs investigating. This example will be updated after fixing the tabs problem.

Reply:
We'll need to fix this once we've had a look at the examples configuration